dubbo-cluster-extensions/dubbo-cluster-specify-address-common/src/main/java/org/apache/dubbo/rpc/cluster/specifyaddress/Address.java

```diff
@@ -16,35 +16,43 @@
  */
 package org.apache.dubbo.rpc.cluster.specifyaddress;
 
-import org.apache.dubbo.common.URL;
-
 import java.io.Serializable;
 import java.util.Objects;
 
 public class Address implements Serializable {
+
+    public static final String name = "specifyAddress";
+
     // ip - priority: 3
     private String ip;
 
     // ip+port - priority: 2
     private int port;
 
     // address - priority: 1
-    private URL urlAddress;
+    private Object urlAddress;
```

Comment:
```suggestion
    private URL urlAddress;
```

Since dubbo is added to dependencies, `URL` can be depend on here.